src/main/kotlin/no/nav/syfo/client/pdl/domain/PdlResponse.kt

```diff
@@ -43,28 +53,73 @@ data class PersonNavn(
 )
 
 data class Adressebeskyttelse(
-    val gradering: Gradering
+    val gradering: Gradering,
 ) : Serializable
 
 enum class Gradering : Serializable {
     STRENGT_FORTROLIG_UTLAND,
     STRENGT_FORTROLIG,
-    FORTROLIG
+    FORTROLIG,
+    UGRADERT, // Ugraderte personer kan også ha tomt felt
 }
 
 data class PdlError(
     val message: String,
     val locations: List<PdlErrorLocation>,
     val path: List<String>?,
-    val extensions: PdlErrorExtension
+    val extensions: PdlErrorExtension,
 )
 
 data class PdlErrorLocation(
     val line: Int?,
-    val column: Int?
+    val column: Int?,
 )
 
 data class PdlErrorExtension(
     val code: String?,
-    val classification: String
+    val classification: String,
 )
+
+fun PdlHentPerson.toPersonName(): String? {
+    val navn = this.hentPerson?.navn?.first()
+
+    return if (navn?.fornavn.isNullOrEmpty() || navn?.etternavn.isNullOrEmpty()) {
+        null
+    } else {
+        "${navn?.fornavn}${getMellomnavn(navn?.mellomnavn)} ${navn?.etternavn}"
+    }
+}
+
+private fun getMellomnavn(mellomnavn: String?): String {
+    return if (mellomnavn !== null) " $mellomnavn" else ""
+}
+
+fun PdlHentPerson.toPersonAdress(): String? {
+    if (this.isNotGradert()) {
+        val vegadresse = this.hentPerson?.bostedsadresse?.first()?.vegadresse
+        if (vegadresse != null) {
+            val adressenavn = vegadresse.adressenavn
+            val husnummer = vegadresse.husnummer ?: ""
+            val husbokstav = vegadresse.husbokstav ?: ""
+            val postnummer = if (!vegadresse.postnummer.isNullOrEmpty()) {
+                ", ${vegadresse.postnummer}"
+            } else {
+                ""
+            }
+
+            log.warn("$adressenavn ${husnummer}${husbokstav}, $postnummer")
+            return "$adressenavn ${husnummer}${husbokstav}$postnummer"
+        }
+    }
+    log.info("Can not get person's address due to adressebeskyttelse")
+    return null
+}
+
+fun PdlHentPerson.isNotGradert(): Boolean {
```

Comment:
Kan vurdere å slå sammen til 

fun PdlHentPerson.isNotGradert(): Boolean {
    val graderingName = this.hentPerson?.adressebeskyttelse?.firstOrNull()?.gradering?.name
    return graderingName == null || graderingName == Gradering.UGRADERT.name
}